src/components/game-field/game-field.css

```diff
@@ -0,0 +1,17 @@
+.game-field {
+  display: flex;
+  flex-flow: column nowrap;
+  align-items: center;
+  height: 100%;
+  min-height: 0;
+}
+
+.game-field__cards {
+  flex: 0 1 calc(var(--vertical-cards-count) * var(--game-card-total-height));
+  display: grid;
```

Comment:
flex + grid?

Reply:
The parent component already uses 'display: flex', so these flex settings are meant for the child component. Perhaps it would be better to pass a class name with flex settings through the props.